py3status/modules/backlight.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""
+Display the current screen backlight level.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (default: 10s)
+    device: The backlight device (default: "acpi_video0")
+            If you are unsure try: `ls /sys/class/backlight`
+    color:  The text color (default: "#FFFFFF")
+
+Format status string parameters:
+    {level} brightness
+
+@author Tjaart van der Walt (github:tjaartvdwalt)
+@license BSD
+"""
+
+from __future__ import division
+
+import subprocess
+from time import time
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 10
+    device = "acpi_video0"
+    format = u'☼: {level}%'
+    color = '#FFFFFF'
```

Comment:
ultrabug is keen on these being in alphabetical order (same with in the docstring)

With the color, I'd be inclined to have the default as `None` and only set it in the responsive if the user specifies one.
As it is #279 will move the color setting out of the modules anyway so I'd not worry too much.